server/routerlicious/packages/lambdas/src/nexus/connect.ts

```diff
@@ -334,7 +336,11 @@ async function checkClusterDraining(
 			...properties,
 			tenantId: message.tenantId,
 		});
-		throw new NetworkError(503, "Cluster is not available. Please retry later.");
+		const error = createFluidServiceNetworkError(503, {
+			message: "Cluster is not available. Please retry later.",
```

Comment:
Message says "retry later", but there is no `retryAfter` value. Should there be?